test/mbd_json.test.c

```diff
@@ -57,513 +58,526 @@ static int test_primitive_object(void) {
         const uint8_t json[] = "{}";
         int result = parse_json(json, sizeof(json), &data);
 
-        check(result == JSON_RESULT_VALUE);
-        check(data.depth == 0);
-        check(data.type == JSON_OBJECT_START);
-        check(data.value.start == 0);
-        check(data.value_counter[0] == 0);
+        TEST_ASSERT_EQUAL( JSON_RESULT_VALUE, result);
```

Comment:
`TEST_ASSERT_EQUAL( JSON_RESULT_VALUE, result)` without the space after the `(` please.